integration_testing/features/cancelling-the-copying-of-an-item.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Cancelling the copying of an item
+	
+	Background:
+		Given I am signed into Kolibri Studio as an admin user

```

Comment:
This is probably an accidental copy/paste from scenarios in Kolibri, and you can delete the `as an admin user` part. Being an `admin` user does not have the same implications in Studio, and makes no sense for the scenarios in this batch (we do no yet have written any for the `Administration` dashboard).